test/e2e-tests/support/client_set.go

```diff
@@ -0,0 +1,34 @@
+package support
+
+import (
+	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+	"os"
+	"path/filepath"
+)
+
+func ClusterClientSet() (*kubernetes.Clientset, error) {
```

Comment:
I am thinking how this will work for two clusters, each having a different KubeConfig, that a test developer would like to use both in a single test? 

Maybe there could be a variant of this function with a single parameter allowing to specify the KubeConfig location (or the name of the env. var to use instead of `KUBECONFIG`)?

On the other hand, this is a nice start and it can be extended later when we know more what is needed regarding two cluster support.

Reply:
Would this be something we are looking for? Because if we are switching contexts then only one kubeconfig should be needed right? You just need to log into them both

Reply:
> On the other hand, this is a nice start and it can be extended later when we know more what is needed regarding two cluster support.

+1 to this.  Right now the existing functionality is focused on providing the core of a go test framework to interact with an kubernetes cluster.  As we plan new features for this repo then we can design the test framework in parallel.

Reply:
Yes, there are basically two options:
* switch contexts
* 2 kubeconfig files
We will need to find out what is more convenient. It will depend on the output of the provisioning job, when it outputs 2 user/password pairs, we would probably use one kubeconfig. If it outputs 2 kubeconfig files, it might be easier for us to reuse these 2 kubeconfig files.

Wdyt @winklerm?

Reply:
> +1 to this. Right now the existing functionality is focused on providing the core of a go test framework 

Sounds good